tests/Integration/StatementsCrudTest.php

```diff
@@ -0,0 +1,276 @@
+<?php
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Exception\InvalidArgumentException;
+use Google\Cloud\Core\Exception\BadRequestException;
+use Oat\DbalSpanner\SpannerConnection;
+use Oat\DbalSpanner\SpannerDriver;
+use Oat\DbalSpanner\SpannerPlatform;
+
+class StatementsCrudTest extends \PHPUnit\Framework\TestCase
+{
+    protected const INSTANCE_NAME = 'php-dbal-tests';
+    protected const DATABASE_NAME = 'spanner-test';
+
+    /** @var SpannerConnection */
+    protected $connection;
+
+    /** @var int */
+    protected $now = 1558344254;
```

Comment:
https://carbon.nesbot.com/docs/ usage may make the test a bit nicer imo

Reply:
I totally agree on using Carbon when something is actually done with the date. Here we don't do anything but storing and retrieving it within a database row, this is why I arbitrarily fixed it to whatever. Carbon use here would be overkill, what do you think?

Reply:
maybe if this is the only use case here, maybe overkill, agreed.
I let you judge to use it or not, both styles are ok for me :)